spec/integration_spec.rb

```diff
@@ -16,33 +16,33 @@ class Comment < ActiveRecord::Base; end
   end
 
   describe 'logging' do
-    context 'when the migration logging is enabled' do
+    context 'when the migration logging is disabled' do
       around(:each) do |example|
         original_verbose = ActiveRecord::Migration.verbose
-        ActiveRecord::Migration.verbose = true
+        ActiveRecord::Migration.verbose = false
         example.run
         ActiveRecord::Migration.verbose = original_verbose
       end
 
       it 'sends the output to the stdout' do
         expect do
           ActiveRecord::Migrator.new(direction, migration_fixtures, 1).migrate
-        end.to output.to_stdout
+        end.to_not output.to_stdout
       end
     end
 
-    context 'when the migration logging is disabled' do
+    context 'when the migration logging is enabled' do
       around(:each) do |example|
         original_verbose = ActiveRecord::Migration.verbose
-        ActiveRecord::Migration.verbose = false
+        ActiveRecord::Migration.verbose = true
         example.run
         ActiveRecord::Migration.verbose = original_verbose
       end
 
       it 'sends the output to the stdout' do
         expect do
           ActiveRecord::Migrator.new(direction, migration_fixtures, 1).migrate
-        end.to_not output.to_stdout
+        end.to output.to_stdout
```

Comment:
what does exactly happen with these context? I'm not sure who but someone had also a problem with them. We should create an issue for it.